server/plugin/webhook.go

```diff
@@ -476,35 +503,46 @@ func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 		return
 	}
 
-	issueTemplate := ""
-	switch action {
-	case actionOpened:
-		issueTemplate = "newIssue"
+	newIssueMessage, err := renderTemplate("newIssue", event)
+	if err != nil {
+		p.API.LogWarn("Failed to render newIssueMessage template", "error", err.Error())
+		return
+	}
 
-	case actionClosed:
-		issueTemplate = "closedIssue"
+	newIssueCollapsedMessage, err := renderTemplate("newIssueCollapsed", event)
+	if err != nil {
+		p.API.LogWarn("Failed to render newIssueCollapsed template", "error", err.Error())
+		return
+	}
 
-	case actionReopened:
-		issueTemplate = "reopenedIssue"
+	closedIssueMessage, err := renderTemplate("closedIssue", event)
+	if err != nil {
+		p.API.LogWarn("Failed to render closedIssue template", "error", err.Error())
+		return
+	}
 
-	case actionLabeled:
-		issueTemplate = "issueLabelled"
+	reopenedIssueMessage, err := renderTemplate("reopenedIssue", event)
+	if err != nil {
+		p.API.LogWarn("Failed to render reopenedIssue template", "error", err.Error())
+		return
+	}
 
-	default:
+	issueLabelledMessage, err := renderTemplate("issueLabelled", event)
+	if err != nil {
+		p.API.LogWarn("Failed to render issueLabelled template", "error", err.Error())
 		return
 	}
 
-	renderedMessage, err := renderTemplate(issueTemplate, event)
+	issueLabelledCollapsedMessage, err := renderTemplate("issueLabelledCollapsed", event)
 	if err != nil {
-		p.API.LogWarn("Failed to render template", "error", err.Error())
+		p.API.LogWarn("Failed to render issueLabelled template", "error", err.Error())
 		return
 	}
-	renderedMessage = p.sanitizeDescription(renderedMessage)
 
 	post := &model.Post{
 		UserId:  p.BotUserID,
 		Type:    "custom_git_issue",
-		Message: renderedMessage,
+		Message: "",
```

Comment:
We don't need to place this here, as Go will use the empty string by default. If this field was a pointer, it would be possible to be `nil`, but in this case it's guaranteed to be a valid string, defaulting to the empty string.

Another spot above also uses this pattern and should be changed to let the struct implicitly set its `Message`